pyiron/sphinx/base.py

```diff
@@ -662,37 +899,66 @@ def _set_kpoints(
         manual_kpoints=None,
         weights=None,
         reciprocal=True,
+        kpoints_per_angstrom=None,
         n_path=None,
         path_name=None,
     ):
         """
         Function to setup the k-points for the Sphinx job
 
         Args:
-            reciprocal (bool): Tells if the supplied values are in reciprocal (direct)
-                               or cartesian coordinates (in reciprocal space) (not implemented)
-            weights (list): Manually supplied weights to each k-point in case of the manual mode (not implemented)
+            reciprocal (bool): Tells if the supplied values are in
+                               reciprocal (direct) or cartesian coordinates
+                               (in reciprocal space) (not implemented)
+            weights (list): Manually supplied weights to each k-point in
+                            case of the manual mode (not implemented)
             manual_kpoints (list): Manual list of k-points (not implemented)
-            symmetry_reduction (bool): Tells if the symmetry reduction is to be applied to the k-points
+            symmetry_reduction (bool): Tells if the symmetry reduction is
+                                       to be applied to the k-points
             scheme (str): Type of k-point generation scheme ('MP' or 'Line')
             mesh (list): Size of the mesh (in the MP scheme)
-            center_shift (list): Shifts the center of the mesh from the gamma point by the given vector
-            n_trace (int): Number of points per trace part for line mode
-            path_name (str): Name of high symmetry path used for band structure calculations.
+            center_shift (list): Shifts the center of the mesh from the
+                                 gamma point by the given vector
+            kpoints_per_angstrom (float): Number of kpoint per angstrom
+                                          in each direction
+            n_path (int): Number of points per trace part for line mode
+            path_name (str): Name of high symmetry path used for band
+                             structure calculations.
         """
         if not isinstance(symmetry_reduction, bool):
             raise ValueError("symmetry_reduction has to be a boolean")
         if manual_kpoints is not None:
             raise ValueError("manual_kpoints is not implemented in SPHInX yet")
         if weights is not None:
-            raise ValueError("manual weights are not implmented in SPHInX yet")
+            raise ValueError(
+                "manual weights are not implmented in SPHInX yet"
+                )
+
```

Comment:
Yeah I guess it's just the Pyiron wrapper, so I changed the error messages to say "not implemented in Pyiron for SPHInX".

Implementing this would be another good pull request for later.

Reply:
new error message fixes my concern, but good idea to put this as an open issue separate from this pull request. Let's discuss tomorrow with Felix - it might overlap with the generic k-point path stuff...